erpnext/selling/doctype/sales_order/sales_order.js

```diff
@@ -87,17 +87,13 @@ frappe.ui.form.on("Sales Order", {
 				frm.events.get_items_from_internal_purchase_order(frm);
 			}
 
-			if (frm.is_new()) {
+			if (frm.doc.docstatus === 0) {
```

Comment:
@s-aga-r Was the change from checking `frm.is_new()` (`frm.doc.__islocal`) to `frm.doc.docstatus` intentional or accidental?